module/applications/srd-compendium.mjs

```diff
@@ -0,0 +1,91 @@
+Hooks.once("ready", () => {
+  const pack = game.packs.get("dnd5e.rules");
+  pack.apps = [new SRDCompendium(pack)];
+});
```

Comment:
I would rather we put this code alongside our other init and ready behaviors in the root dnd5e module file.